web/src/app/modules/shared/components/presentation/tabs/tabs.component.html

```diff
@@ -1,8 +1,13 @@
 <div class="clr-row clr-align-items-center clr-justify-content-between">
-  <div class="clr-col-10">
+  <div class="clr-col-6">
     <app-breadcrumb [path]="title"> </app-breadcrumb>
   </div>
-  <div class="clr-col-2">
+  <div class="clr-col-6">
+    <div *ngIf="titleComponents?.length > 1" class="title-container">
+      <div *ngFor="let component of titleComponents; let i = index; trackBy: identifyPath" class="item">
+        <app-view-container [view]="component"></app-view-container>
+      </div>
+    </div>
     <div *ngIf="buttonGroup" class="action-buttons">
```

Comment:
This button group can be removed in favor of using title components altogether. `AddButton` in `component.go` will need to be updated to add a button into title components and buttonGroup can be removed.